lxd/instance/drivers/driver_lxc.go

```diff
@@ -2235,6 +2241,29 @@ func (d *lxc) startCommon() (string, []func() error, error) {
 		}
 	}
 
+	if len(cdiHookCmds) > 0 {
+		postStartHooks = append(
```

Comment:
In the context of OCI there are a number of points at which hooks can be invoked -- relative to runtime and container lifecycle -- does LXD make this distinction, or is there only one set of hooks?

Reply:
Yes we have the same approach: there are only a few LXD runtime hooks : `onStart`, `postStart`, `onStop` and `onStopNS` IIRC. The container lifecycle hooks (more numerous) allow more fine grained operations and are defined here: https://linuxcontainers.org/lxc/manpages/man5/lxc.container.conf.5.html#:~:text=ids%20to%20map.-,CONTAINER%20HOOKS,-Container%20hooks%20are